### ISSUE TITLE
Loading Content in a Game

### ISSUE THREAD

**Issue by racouster** [rbwhitaker/rbwhitaker.wikidot.com#12]
The following text looks incomplete:

[monogame-managing-content](http://rbwhitaker.wikidot.com/monogame-managing-content)
Section: Loading Content in a Game

"We still need to load the game-ready asset into the game itself. The details of how to use these assets will be covered in more detail in future tutorials, but they all follow a common pattern: you use a ContentManager instance to"
Missing content?
"In the coming tutorials, we're going to load a lot of content in our games. So a simple example now will help illuminate the pathway."

Screenshot:
![Screenshot 2024-07-14 111551](https://github.com/user-attachments/assets/4f239a0a-691a-4f36-8cbd-ce93749ece59)

**Comment by rbwhitaker** [comment]
I agree. Something's missing. I'll get it fixed. Thanks for reporting it!
